### DISCUSSION TITLE
What folder do you guys save this in on MacOS?

### DISCUSSION THREAD

**Discussion by pendolino** [FilipePS/Traduzir-paginas-web#416] [Q&A]
It's currently running out of my downloads folder but how do I get Brave to import those files? If I remove them to a different location it does not run and I have to reinstall or replace the files back.

**Comment by FilipePS** [comment]
Simply put the extension in another folder, so you don't delete files often.
On windows it could be, for example, in the Documents folder, on Mac OS, I don't know.
